apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-notes-panel.presenter.tsx

```diff
@@ -8,20 +7,12 @@ interface SelfNotesPanelProps {
     notes: SelfConnectionNote[];
 }
 
-export const SelfNotesPanelPresenter = ({
-    compositeIdentifier: { type: resourceType, id },
-    notes,
-}: SelfNotesPanelProps): JSX.Element => (
+export const SelfNotesPanelPresenter = ({ notes }: SelfNotesPanelProps): JSX.Element => (
```

Comment:
Would make sense to rename to something like `SelfNotesListPresenter` as it’s no longer a panel itself.  Drawer is declared in our new `CategorizablePageLayout`.  Not sure if "List" would be the right thing either.

Reply:
`SelfNotesForResourcePresenter`?

Reply:
Ya, that sounds good.